pkg/interfaces/contracts/vault/ICompositeLiquidityRouter.sol

```diff
@@ -39,7 +40,8 @@ interface ICompositeLiquidityRouter {
      */
     function addLiquidityUnbalancedToERC4626Pool(
         address pool,
-        uint256[] memory exactUnderlyingAmountsIn,
+        bool[] memory useWrappedTokens,
```

Comment:
I think `useWrappedTokens` could be a good name. It could be `isWrappedToken` (in the style of `wethIsEth`), but the point of this is to allow having wrapped tokens that are treated as standard tokens, so the flag does indicate whether the token should be "used" as a wrapped token.

However, it seems like the sense of the boolean is reversed from what I would expect. To me "useWrapped" means "use the token as an ERC4626"; i.e., wrap/unwrap. But from the code, it looks like it means the opposite: if the flag is set, consider it a regular token.

Maybe `wrapUnwrapTokens` would be clearer? If true, wrap/unwrap; otherwise, consider it a standard token.